shell/browser/ui/inspectable_web_contents.cc

```diff
@@ -908,8 +908,9 @@ void InspectableWebContents::HandleMessageFromDevToolsFrontend(
     params = &empty_params;
   }
   int id = message.FindIntKey(kFrontendHostId).value_or(0);
-  base::ListValue* params_list = nullptr;
-  params->GetAsList(&params_list);
+  std::vector<base::Value> params_list;
+  if (params)
+    params_list = std::move(*params).TakeList();
```

Comment:
It's not obvious to me that it's OK to `std::move` params here. @jkleinsc am I missing something?

Reply:
@nornagon I duplicated what was done in the CL: https://chromium-review.googlesource.com/c/chromium/src/+/3000931; specifcally:
https://chromium-review.googlesource.com/c/chromium/src/+/3000931/5/chrome/browser/devtools/devtools_ui_bindings.cc, though it looks like there is some additional code we can drop as well.